tests/python/unittest/test_target_codegen_spirv.py

```diff
@@ -0,0 +1,78 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import tvm
+import tvm.testing
+from tvm import te
+from tvm.topi.math import cast
+import numpy as np
+
+
+tx = te.thread_axis("threadIdx.x")
+ty = te.thread_axis("threadIdx.y")
```

Comment:
it is better to keep thread axis construction in local functions

Reply:
sure no problem, it was just copy-pasted from other test https://github.com/apache/tvm/blob/e561007f0c330e3d14c2bc8a3ef40fb741db9004/tests/python/unittest/test_target_codegen_rocm.py#L23-L26

fixed.